src/photos.cs

```diff
@@ -944,9 +993,13 @@ interface PHImageManager {
 		CGSize MaximumSize { get; }
 
 		[iOS (9,1)]
-		[NoMac]
+		[Mac (10,15, onlyOn64: true)]
 		[Export ("requestLivePhotoForAsset:targetSize:contentMode:options:resultHandler:")]
 		int /* PHImageRequestID = int32_t */ RequestLivePhoto (PHAsset asset, CGSize targetSize, PHImageContentMode contentMode, [NullAllowed] PHLivePhotoRequestOptions options, PHImageManagerRequestLivePhoto resultHandler);
+
+		[TV (13,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Export ("requestImageDataAndOrientationForAsset:options:resultHandler:")]
+		int RequestImageDataAndOrientationForAsset (PHAsset asset, [NullAllowed] PHImageRequestOptions options, PHImageManagerRequestImageDataHandler resultHandler);
```

Comment:
I think `RequestImageDataAndOrientation` sounds better:

```suggestion
		int RequestImageDataAndOrientation (PHAsset asset, [NullAllowed] PHImageRequestOptions options, PHImageManagerRequestImageDataHandler resultHandler);
```